pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.black]
+line-length = 79
+target-version = ["py38"]
+include = '\.py?$'
+exclude = '''
```

Comment:
You will have to use `force-exclude` here, otherwise black is going to ignore these. See also https://github.com/rapidsai/cudf/pull/12036 .

Reply:
thanks for the pointer!  I've updated here